package.json

```diff
@@ -99,7 +99,6 @@
     "react-hot-loader": "4.12.16",
     "react-keydown": "^1.9.7",
     "react-lazyload": "^2.3.0",
-    "react-lottie": "^1.2.3",
```

Comment:
Have to make sure we add this back into getsentry in that case, since it's needed for the animation in the upsell modal.

Reply:
Ah good thinking. It's already in getsentry separately https://github.com/getsentry/getsentry/blob/master/package.json#L37